themes/cardinal_service/lib/components/minimal-card/minimal_card.html.twig

```diff
@@ -0,0 +1,16 @@
+<div{{ attributes.addClass('su-minimal-card') }}>
+  <div class="flex-container su-card__contents">
+    {% if image %}
+      <div class="flex-8-of-12">
+        <div class="su-card-img">
+          {{ image|render|striptags('<drupal-render-placeholder> <img> <picture> <source>')|raw }}
+        </div>
+      </div>
+    {% endif %}
+    <div class="{% if image|render|striptags('<drupal-render-placeholder> <img> <picture> <source>')|trim  %}flex-4-of-12{% endif %}">
```

Comment:
what can be a little better on performance is to set a variable at the top of the file. 

`{% set image = image|render|striptags('<drupal-render-placeholder> <img> <picture> <source>')|trim %}`
then you can render out the image and check it in different places.